### ISSUE TITLE
[Bug]: Double scroll on file editor component hides save button

### ISSUE THREAD

**Issue by malhotra5** [All-Hands-AI/OpenHands#5391]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Describe the bug and reproduction steps

Theres a double scroll on the file editor component. 

1. You can scroll the code file
2. You can also scroll the code file's container, which ends up hiding the "save" button

Once you scroll the container down, you can't scroll back up 🙂

### OpenHands Installation

app.all-hands.dev

### OpenHands Version

latest

### Operating System

MacOS

### Logs, Errors, Screenshots, and Additional Context

https://github.com/user-attachments/assets/ff9c337a-86bb-47f2-9940-f39e5ce6faba

**Comment by dannypaz** [comment]
@malhotra5 Q for you. What is the expected functionality? Should we remove the ability to scroll on the files container OR do we want to handle scrolling updown on that window?

**Comment by malhotra5** [comment]
Hi @dannypaz! I'm not sure if I understand the question, but let me rephrase the requirements. We want to be able to independently scroll 

- The workspace file directory
- The code file

We don't want to scroll 
- The parent component of the code file (because it hides the save/discard buttons from view and can't be recovered as shown in the video). Another way of saying this is the save/discard buttons should be a sticky component that don't leave the view even when the file directory or code files are being scrolled

**Comment by dannypaz** [comment]
@malhotra5 thank you. that is what I was wondering. I will take a stab at it

**Comment by malhotra5** [comment]
Closing as resolved by #5488
